config/display.conf

```diff
@@ -0,0 +1,17 @@
+// Configuration files are installed in /etc/gamescope.d/*.conf
+{
+    "display_configuration": {
+        "VLV": {
+            "0x3003" : {
+                "refresh_rates": [ 45, 47, 48, 49, 50, 51, 53, 55, 56, 59,
+                                   60, 62, 64, 65, 66, 68, 72, 73, 76, 77,
+                                   78, 80, 81, 82, 84, 85, 86, 87, 88, 90 ]
+            },
+            "0x3004" : {
+                "refresh_rates": [ 45, 47, 48, 49, 50, 51, 53, 55, 56, 59,
+                                   60, 62, 64, 65, 66, 68, 72, 73, 76, 77,
+                                   78, 80, 81, 82, 84, 85, 86, 87, 88, 90 ]
+            }
+        }
+    }
+}
```

Comment:
Should we add an example_config.json to the install output? Sort of as a config documentation with every field commented out.

Reply:
Another option is installing it as `config.json.sample` so it would be ignored by default because of the extension (that's how scripts in `.git/hooks` work).